source/brailleDisplayDrivers/handyTech.py

```diff
@@ -373,6 +375,15 @@ def display(self, cells: List[int]):
 		super(StatusCellMixin, self).display(cells)
 
 
+class ActiveSplitMixin(object):

```

Comment:
All python 3 classes inherit object by default, if you want to make a mixin like structure I encourage using [abstract base classes](https://docs.python.org/3/library/abc.html)

```suggestion
class ActiveSplitMixin:
```